base/src/com/google/idea/blaze/base/command/info/BlazeInfoProvider.java

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
I'm not sure we can cache this between runs. Command-line flags affect the outcome (e.g. `--server_javabase`), which can be in any of multiple locations and bazelrc files. Caching for a single run seems okay, since a lot of the flags that change the info output are startup flags and would invalidate most of the sync anyway.

Reply:
I've updated the PR to invalidate after each sync.